doc/source/examples/cfd-dem/square-fluidized-bed/square-fluidized-bed.rst

```diff
@@ -0,0 +1,392 @@
+
+.. code-block:: text
+
+  dem_3d packing_in_circle.prm
+
+or in parallel (where 8 represents the number of processors)
+
+.. code-block:: text
+
+  mpirun -np 8 dem_3d packing_in_circle.prm
+
+Lethe will generate a number of files. The most important one bears the extension ``.pvd``. It can be read by popular visualization programs such as `Paraview <https://www.paraview.org/>`_. 
+
+
+.. note:: 
+    The vtu files generated by Lethe are compressed archives. Consequently, they cannot be postprocessed directly. Although they can be easily post-processed using Paraview, it is sometimes necessary to be able to work with the raw data. The python library `PyVista <https://www.pyvista.org/>`_  allows us to do this. 
+
+After the particles have been packed inside the square bed, it is now possible to simulate the fluidization of particles.
+
+The CFD simulation is to be carried out using the packed bed simulated in the previous step. We will discuss the different parameter file sections. The mesh section is identical to that of the DEM so it will not be shown here.
+
+Simulation control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation was run for 1 s with a time step of 0.002 s. The time scheme chosen for the simulation is first order backward difference method (BDF1). The simulation control section is shown:
+
+.. code-block:: text
+
+    subsection simulation control
+        set method                          = bdf1
+        set number mesh adapt               = 0
+        set output name                     = result_
+        set output frequency	  	    = 10
+        set startup time scaling            = 0.6
+        set time end                        = 1
+        set time step                       = 0.002
+        set subdivision                     = 1 
+        set log precision                   = 10 
+        set output path                     = ./output/
+end
+
+
+
+Physical properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The physical properties subsection allows us to determine the density and viscosity of the fluid. We choose a density of 1 and viscosity of 0.00001 as to simulate the flow of air. 
+
+.. code-block:: text
+
+    subsection physical properties
+        subsection fluid 0
+            set kinematic viscosity            = 0.00001
+            set density                        = 1
+          end
+    end
+
+
+Initial conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the initial conditions, we choose zero initial conditions for the velocity. 
+
+.. code-block:: text
+
+    subsection initial conditions
+        set type = nodal
+        subsection uvwp
+            set Function expression = 0; 0; 0; 0
+        end
+    end
+ 
+
+Boundary conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the boundary conditions, we choose a slip boundary condition on the walls of the cylinder (ID = 0) and an inlet velocity of 0.2 m/s at the lower face of the bed (ID = 2).
```

Comment:
Can you adapt the text to the fact that this is a square bed? There seems to be a confusion because you talk of a cylinder :)

Reply:
It seems these changes I already did but were not pushed correctly. Done